modules/ROOT/pages/inline-css.adoc

```diff
@@ -0,0 +1,116 @@
+= Inline CSS plugin
+:navtitle: Inline CSS
+:description: takes {productname} documents and outputs HTML formatted according to email-friendly HTML best practices (eg inlined CSS).
+:description_short: Inlines {productname} CSS for better presentation in email.
+:keywords: plugin, inlinecss, email friendly, email, output, inline, css, post-process, html
+:pluginname: Inline CSS
+:plugincode: inlinecss
+:plugincategory: premium
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-6.3v.adoc[]
+
+As is normal and long-time best practice, {productname} documents keep CSS and HTML separate. This practice is not, however, email-friendly.
```

Comment:
I'm not sure if we're limiting the use case of this plugin too much to just email. Probably not, but just thought I'd ask. It might have other uses. Can't think of any off-hand, though.

Reply:
@ephox-mogran Epic title has "Email Friendly Output"
Epic Link: https://ephocks.atlassian.net/browse/EPIC-31 
Epic Title : Inline CSS (Email Friendly Output))